pallets/manta-sbt/src/lib.rs

```diff
@@ -343,17 +373,28 @@ pub mod pallet {
         #[pallet::call_index(1)]
         #[pallet::weight(<T as pallet::Config>::WeightInfo::reserve_sbt())]
         #[transactional]
-        pub fn reserve_sbt(origin: OriginFor<T>) -> DispatchResult {
+        pub fn reserve_sbt(origin: OriginFor<T>, reservee: Option<T::AccountId>) -> DispatchResult {
             let who = ensure_signed(origin)?;
+            // Use reservee account, if None then use account whom signed transaction
+            let reserve_account = reservee.unwrap_or(who.clone());
+
+            let allowlist_account = AllowlistAccount::<T>::get();
```

Comment:
is it ok to add another account to avoid nonce issue as much as possible.

Reply:
yeah that's a good idea